config/routes.rb

```diff
@@ -4,9 +4,17 @@
   get 'search', to: 'pages#search'
   resources :articles
   get 'signup', to: 'users#new'
-  resources :users, except: [:new]
+  
+  # resources :users, except: [:new]
+  resources :users do
+    member do
+      get :confirm_email
```

Comment:
Das ist inhaltlich nicht ganz richtig, weil wir dadurch die Route in der Art `/users/:id/confirm_email` erhalten.
Du schreibst aber oben `confirm_email_user_url(@user.confirm_token)`. Also das funktioniert dann schon, aber wir geben den token als id rein. Das ist für andere Entwickler*innen verwirrend.

Ich denke, wir brauchen entweder eine Collection, ich weiß nicht, ob sich das so schreiben lässt:
```
collection do
  get 'confirm_email/:token',
end
```
oder definieren die Route einfach komplett neu, also außerhalb der users Ressource
```get 'users/confirm_email/:token', to: ...```

Reply:
Wenn ich eine neue Route schreibe, dann so?
`get 'users/confirm_email/:token', to: users#confirm_email` ?

Reply:
:+1: